docs/explanation/remote.rst

```diff
@@ -14,82 +14,96 @@ This is especially useful on headless devices.
 Comparison with SSH
 ===================
 
-It's easy to lose SSH connection with the DUT, and if the device doesn't offer
-screen-like functionality then the Checkbox session has to be started over.
+It's easy to lose SSH connection with the :term:`Agent`, and if the device
```

Comment:
The original text was about "losing" local checkbox session. This doesn't translate directly into Agent.